api/src/paths/project/{projectId}/survey/{surveyId}/critters/index.ts

```diff
@@ -235,3 +321,38 @@ export function addCritterToSurvey(): RequestHandler {
     }
   };
 }
+
+export function updateSurveyCritter(): RequestHandler {
+  return async (req, res) => {
+    const user: ICritterbaseUser = {
+      keycloak_guid: req['system_user']?.user_guid,
+      username: req['system_user']?.user_identifier
+    };
+    const surveyId = Number(req.params.surveyId);
+    const connection = getDBConnection(req['keycloak_token']);
+    const surveyService = new SurveyCritterService(connection);
+    const cb = new CritterbaseService(user);
+    try {
+      await connection.open();
+      if (!req.body.update.critter_id) {
```

Comment:
This check can be removed if using the `{critter_id}.ts` file setup. As the endpoint cannot be reached without the param